src/fu-config.c

```diff
@@ -241,6 +243,19 @@ fu_config_reload(FuConfig *self, GError **error)
 	if (host_bkc != NULL && host_bkc[0] != '\0')
 		self->host_bkc = g_steal_pointer(&host_bkc);
 
+	/* get trusted uids */
+	g_ptr_array_set_size(self->trusted_uids, 0);
+	uids = g_key_file_get_string_list(keyfile,
+					  "fwupd",
+					  "TrustedUids",
+					  NULL, /* length */
+					  NULL);
+	if (uids != NULL) {
+		for (guint i = 0; uids[i] != NULL; i++) {
+			g_ptr_array_add(self->trusted_uids, g_strdup(uids[i]));
```

Comment:
I think it makes sense to do the fu_common_strtoull() here and return a GArray of guint32 (or is guint64 more appropriate?) rather than doing all the string parsing again and again.

Reply:
fixed